elastic/eui
# [EUI+] update homepage component illustrations

## Summary

This PR updates the "highlights" section of the new docs website based on design iterations. It updates the content and adds new illustration svg files.

![Screenshot 2024-08-08 at 19 36 25](https://github.com/user-attachments/assets/91c8060a-fe02-44fa-a3cd-511019f215ad)

![Screenshot 2024-08-08 at 19 36 22](https://github.com/user-attachments/assets/fdee22ae-820c-4dc4-9feb-725ceb7c584a)

## QA

- review the deployed [homepage](https://eui.elastic.co/pr_7952/new-docs/) and verify it matches design, loads and displays the svgs correctly and colors are properly update on color mode change
